src/components/Containers/TopSection/TopSection.jsx

```diff
@@ -0,0 +1,139 @@
+import React from "react";
+import styled from "styled-components";
+
+import TitleGraphic from "../../../assets/home/top-section-graphic.svg";
+import { media } from "../../../utils";
+import { Text } from "../../Texts";
+import { Button } from "../../Inputs";
+
+export const TopSection = ({ ...props }) => (
+  <MainContainer {...props}>
+    <FirstColumn>
+      <JosefinTitle size="4.5em" bold="true">
+        Prospective Medical Professionals
+      </JosefinTitle>
+      <SText size="default">
+        A registered charity dedicated to connecting aspiring students to health
+        sciences opportunities
+      </SText>
+      <Button>
+        <ButtonText size="default" bold="true">
+          DONATE
+        </ButtonText>
+      </Button>
+    </FirstColumn>
+    <SecondColumn>
+      <Graphic
+        src={TitleGraphic}
+        alt="Title graphic - person with book, stethoscope, pills and syringe"
+      />
+    </SecondColumn>
+  </MainContainer>
+);
+
+const MainContainer = styled.div`
+  display: flex;
+  margin: 5em;
+  flex-wrap: nowrap;
+  flex-direction: row;
+  ${media(
```

Comment:
I'd actually suggest doing this as early as tablet or at a breakpoint like ~700-800 because there are some weird images (like the svgs is really small) at certain breakpoints.

You also need 
```align-items: center``` or else the items won't be vertically aligned. It works for tablet and mobile for storybook but won't work well for all the breakpoints (you can just stretch the window of storybook to see).

I'd also suggest putting a width on the first column if you don't decide to switch breakpoints because near like 800px it starts being like 90% text and a really small svg